### PR TITLE
Wrap routes in switch

### PR DESCRIPTION
**Pull request by kalilsn**
Turns out the router renders all the matching routes, not just the first or most specific one, if you don't have `<Switch>` around them.

### CODE REVIEW

File: src/Routes.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export function topLevelRoutes(props: RouteComponentProps<void>) {
         <Route exact={true} path="/index.html" component={Home} />
 
         {/* Legacy routes that we want to maintain: */}
-        <Route path="/explore/terms-of-service" render={props1 => <Redirect to="/terms-of-service" />} />
+        <Route path="/explore/terms-of-service" render={() => <Redirect to="/terms-of-service" />} />
```
Comment (kalilsn) [inline src/Routes.tsx]: I didn't actually mean to commit this, but I do think it's slightly clearer!

Comment (dvogel) [reply src/Routes.tsx]: Does it work? I remember trying something similar and being annoyed at an error. I don't remember if it was from tsc, tslint, or something in the browser console though.

File: src/components/PageContent.tsx [hunk 2]
```diff
@@ -23,7 +23,9 @@ export class PageContent extends React.Component<IPageContentProps, {}> {
         return (
             <div className="PageContent">
               <div className="content-main" ref={(loader) => (this.loader = loader)} tabIndex={-1}>
-                {this.props.children}
+                <Switch>
+                  {this.props.children}
+                </Switch>
```
Comment (dvogel) [inline src/components/PageContent.tsx]: Should this `Switch` be here or wrapping the `Routes` themself? What if someone uses `PageContent` with children that don't include a `Route`. Does that work?

File: src/Routes.tsx (hunk 1 above)
Comment (kalilsn) [reply src/Routes.tsx]: It's working fine for me locally! Maybe the error was from trying to use `props`, which throws this tslint error: `Shadowed name: 'props' (no-shadowed-variable)`?

File: src/components/PageContent.tsx (hunk 2 above)
Comment (kalilsn) [reply src/components/PageContent.tsx]: Good point. I think you're right that it should be in Routes.tsx

**Pull request merged**